src/DotVVM.Framework/Controls/GridView.cs

```diff
@@ -187,7 +187,9 @@ private void DataBind(IDotvvmRequestContext context)
                 };
             }
 
-            CreateHeaderRow(context, sortCommand);
+            var header = CreateHeaderRow(context, sortCommand);
```

Comment:
I think that in DotVVM 2.0, it doesn't matter anymore. The reason was that the bindings wouldn't compile when the control wasn't a part of the control tree. 
But UI tests should cover this scenario - we have enough tests on gridview sorting.

Reply:
It still changes the state in which the header row is initialized. I think that it will not break any built-in control, but it's a change in API and could break some third-party controls, maybe our BP or BS ones.

Reply:
I don't think that this change can break anything.
Only usage of formally private method CreateHeaderRow is in another private method DataBind.

This is the minimal subset of changes that we need for BS.
Only other way how to modify header is to copy large chunks of Dotvvm internal code to BS only to do some minor changes.
Or we can try to modify content as it renders, which sounds to me as terrible and error prune solution.

Reply:
Yes, but `GridView.CreateHeaderRow` invokes a virtual method `GridViewColumn.CreateHeaderControls`.